package.json

```diff
@@ -68,8 +67,7 @@
     "ttypescript": "^1.5.8",
     "typechain": "^1.0.3",
     "typechain-target-web3-v1": "^1.0.0",
-    "typescript": "^3.7.2",
-    "typescript-transform-paths": "^1.1.13"
```

Comment:
Build breaks because you removed this plugin

Reply:
yep, saw that! I've removed that by mistake...
I've moved it to dex-js but it needs to be here too